build.sbt

```diff
@@ -7,6 +7,7 @@ skip in publish := true
 lazy val integrationTestSettings: Seq[Def.Setting[_]] = Defaults.itSettings ++ Seq(
   scalaSource in IntegrationTest := baseDirectory.value / "src" / "test" / "scala",
   javaSource in IntegrationTest := baseDirectory.value / "src" / "test" / "java",
+  resourceDirectory in IntegrationTest := baseDirectory.value / "src" / "test" / "resources",
```

Comment:
the it tests didn't actually run for all the projects, as some of them had classpath resources that were not built into it-classes (as it was looking for it resources rather than normal ones).  This fixes that.